core/src/main/java/org/sonarsource/sonarlint/core/ConnectedSonarLintEngineImpl.java

```diff
@@ -122,8 +121,7 @@ public RuleDetails getRuleDetails(String ruleKey) {
     }
   }
 
-  @Override
-  public AnalysisResults analyze(AnalysisConfiguration configuration, IssueListener issueListener) {
+  public AnalysisResults analyze(ConnectedAnalysisConfiguration configuration, IssueListener issueListener) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1161)